### DISCUSSION TITLE
Welcome to Snel Discussions!

### DISCUSSION THREAD

**Discussion by buttercubz** [crewdevio/Snel#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Valexr** [comment]
How I can upgrade Snel to new release – run install.ts script one more time?

**Reply by buttercubz** [reply]
yes, you can upgrade running install.ts again, but an upgrade command would be useful

**Comment by Valexr** [comment]
What is this?
![Screenshot 2021-04-04 at 09 27 22](https://user-images.githubusercontent.com/5656366/113500621-13e5f480-9528-11eb-8c71-5657216e3b94.jpg)
